src/main/java/io/goobox/sync/storj/UploadFileTask.java

```diff
@@ -59,11 +61,26 @@ public void run() {
         logger.info("Uploading file {}", fileName);
 
         final boolean repeat[] = { true };
+        Path tmpDir = null;
+        Path tmpPath = null;
+
+        try {
+            tmpDir = Files.createTempDirectory(Utils.getSyncDir(), ".~");
+            tmpPath = Files.createTempFile(tmpDir, "file", ".tmp");
+            Files.copy(path, tmpPath, StandardCopyOption.REPLACE_EXISTING);
+        } catch (IOException e) {
+            logger.info("file {} removed during temporary file creation?", path);
+            logger.info("", e);
```

Comment:
You can append the exception to the previous log message. See here: https://www.slf4j.org/faq.html#paramException